doc/modules/grid_search.rst

```diff
@@ -654,12 +667,11 @@ entry for :term:`n_jobs`.
 Robustness to failure
 ---------------------
 
-Some parameter settings may result in a failure to ``fit`` one or more folds
-of the data.  By default, this will cause the entire search to fail, even if
-some parameter settings could be fully evaluated. Setting ``error_score=0``
-(or `=np.nan`) will make the procedure robust to such failure, issuing a
-warning and setting the score for that fold to 0 (or `nan`), but completing
-the search.
+Some parameter settings may result in a failure to ``fit`` one or more folds of
```

Comment:
This is not related to dropdowns but that part appears to be outdated as `error_score=np.nan` is now the default; LMK if it should be handled in another issue & pr instead

Reply:
Nice catch! I think it's ok to address the issue here.